index.js

```diff
@@ -318,6 +318,10 @@ class ExecutorQueue extends Executor {
      * @return {Promise}
      */
     async _startFrozen(config) {
+        config.job = {
```

Comment:
usually it's not recommended to modify the params, can use a different variable 🤷‍♀️

Reply:
agree, should probably create a new var and object.assign to combine it

Reply:
Fixed